specification/network/resource-manager/Microsoft.Network/preview/2022-04-01-preview/examples/NetworkManagerAdminRuleCollectionDelete.json

```diff
@@ -5,9 +5,15 @@
     "resourceGroupName": "rg1",
     "networkManagerName": "testNetworkManager",
     "configurationName": "myTestSecurityConfig",
-    "ruleCollectionName": "testRuleCollection"
+    "ruleCollectionName": "testRuleCollection",
+    "force": false
   },
   "responses": {
+    "202": {
+      "headers": {
+        "Location": "https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/SampleRG/providers/Microsoft.Network/networkManagers/TestNM/operationResults/00000000-0000-0000-0000-000000000000?api-version=2022-04-01-preview"
```

Comment:
will the location uri stop at the network manager? or is it specific to the resource (in this case rule collection)